osc-language-configuration.json

```
@@ -29,8 +29,9 @@
     },
     // Code folding
     "folding": {
-    "markers": {
-      "start": "{((macro|trigger):([\\w\\-+]+))|if|else}",
-      "end": "{(endif|end)}"
+        "markers": {
+            "start": "{((macro|trigger):([\\w\\-+]+))|if|else}",
```

I think we should use the same regex used in `increaseIndentPattern`, which also includes some other block names. It doesn't make much difference, it's just for consistency.
```suggestion
            "start": "{(init|frame|frame_ai|if|else|(macro|trigger):([\\w\\-+]+))}",
```